gui/default/syncthing/folder/editIgnoresModalView.html

```diff
@@ -14,12 +14,14 @@
     </dl>
   </div>
   <div class="modal-footer">
-    <div class="pull-left"><span translate>Editing</span> <code>{{currentFolder.path}}{{system.pathSeparator}}.stignore</code></div>
```

Comment:
We are not actually editing the file at path but creating or overwriting it. I consider this important enough to warn the user about it.

The second new string is because when adding a new folder the path is not guaranteed to be set. This results in a message saying you are editing `/.stignore`. This could be gotten rid of by disabling ignores until the configuration is valid (like the "Save" button).